app/views/arts/build/address_confirmation.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="wizard-form">
+  <%= form_for @art, method: :put, url: wizard_path do |f| %>
+    <% if f.object.errors.any? %>
+      <div class="error_messages">
+        <% f.object.errors.full_messages.each do |error| %>
+          <p><%= error %></p>
```

Comment:
Can you give this a class of text-red to have the error messages appearing in the same red as the others ?